README.md

```diff
@@ -364,6 +404,8 @@ If you derived your server public key from ssh, all you need in your configurati
   # sops.defaultSopsFile = "/root/.sops/secrets.yaml";
   sops.defaultSopsFile = ./secrets.yaml;
   sops.secrets.example-key = {};
+  # This is using ssh keys in the age format:
+  sops.age.sshKeyPaths = [ "/etc/ssh/ssh_host_ed25519_key" ];
```

Comment:
Should we just import this by default? Should be quite cheap to do so and than users cannot forget.